syncano/resultset.py

```diff
@@ -0,0 +1,35 @@
+
+
+class ResultSet(object):
+
+    def __init__(self, connection, data, **kwargs):
+        self.connection = connection
+        self.data = data or {}
+        self.result_class = kwargs.get('result_class')
+        self.request_method = kwargs.get('request_method')
+        self.request_path = kwargs.get('request_path')
+        self.request_params = kwargs.get('request_params') or {}
+
+    def __repr__(self):
+        return 'ResultSet for {0} {1}'.format(self.request_method, self.request_path)
+
+    def __str__(self):
+        return 'ResultSet for {0} {1}'.format(self.request_method, self.request_path)
+
+    def __iter__(self):
+        '''Pagination handler'''
+
+        while True:
+            objects = self.data.get('objects')
+            next_url = self.data.get('next')
+            for o in objects:
```

Comment:
Oh, it's o! I would prefer 'object', but well, in python it means something different already ;). So it's a tough case.